pnpm devで依存関係の更新が反映されない

### 💡 Summary

pnpm devだと、例えば packages/misskey-bubble-game を編集したとしても古い内容が使われ続ける
以下を試しても変わらなかった

- pnpm clean
- pnpm build

なお、pnpm build からの pnpm start した場合は正常に読み込まれる(その後 pnpm dev するとやはり古い内容に戻る)

### 🥰 Expected Behavior

更新が反映される

### 🤬 Actual Behavior

更新が反映されない

### 📝 Steps to Reproduce

_No response_

### 💻 Frontend Environment

```markdown
* Model and OS of the device(s):
* Browser:
* Server URL:
* Misskey:
```


### 🛰 Backend Environment (for server admin)

```markdown
* Installation Method or Hosting Service:
* Misskey:
* Node:
* PostgreSQL:
* Redis:
* OS and Architecture:
```


### Do you want to address this bug yourself?

- [ ] Yes, I will patch the bug myself and send a pull request

あと pnpm dev するとパッケージを import してるとこが Cannot find module 'misskey-bubble-game' or its corresponding type declarations みたいになってしまう(VSCode再起動すれば直るけど、pnpm dev するたびに再起動必要というのは意味がなくなってしまっている)

--- a/packages/misskey-bubble-game/package.json
+++ b/packages/misskey-bubble-game/package.json
@@ -2,6 +2,7 @@
 	"type": "module",
 	"name": "misskey-bubble-game",
 	"version": "0.0.1",
+	"types": "./built/dts/index.d.ts",
 	"exports": {
 		".": {
 			"import": "./built/esm/index.js",
@@ -15,7 +16,7 @@
 	"scripts": {
 		"build": "node ./build.js",
 		"build:tsc": "npm run tsc",
-		"tsc": "npm run ts-esm && npm run ts-dts",
+		"tsc": "npm run tsc-esm && npm run tsc-dts",
 		"tsc-esm": "tsc --outDir built/esm",
 		"tsc-dts": "tsc --outDir built/dts --declaration true --emitDeclarationOnly true --declarationMap true",
 		"watch": "nodemon -w src -e ts,js,cjs,mjs,json --exec \"pnpm run build:tsc\"",

--- a/packages/misskey-js/package.json
+++ b/packages/misskey-js/package.json
@@ -19,7 +19,7 @@
 		"ts": "npm run ts-esm && npm run ts-dts",
 		"ts-esm": "tsc --outDir built/esm",
 		"ts-dts": "tsc --outDir built/dts --declaration true --emitDeclarationOnly true --declarationMap true",
-		"watch": "nodemon -w src -e ts,js,cjs,mjs,json --exec \"pnpm run build\"",
+		"watch": "nodemon -w src -e ts,js,cjs,mjs,json --exec \"pnpm run ts\"",
 		"tsd": "tsd",
 		"api": "pnpm api-extractor run --local --verbose",
 		"api-prod": "pnpm api-extractor run --verbose",

--- a/packages/misskey-reversi/package.json
+++ b/packages/misskey-reversi/package.json
@@ -2,6 +2,7 @@
 	"type": "module",
 	"name": "misskey-reversi",
 	"version": "0.0.1",
+	"types": "./built/dts/index.d.ts",
 	"exports": {
 		".": {
 			"import": "./built/esm/index.js",

--- a/scripts/dev.mjs
+++ b/scripts/dev.mjs
@@ -28,19 +28,19 @@ await execa('pnpm', ['build-assets'], {
 	stderr: process.stderr,
 });
 
-await execa('pnpm', ['--filter', 'misskey-js', 'build'], {
+await execa('pnpm', ['--filter', 'misskey-js', 'ts'], {
 	cwd: _dirname + '/../',
 	stdout: process.stdout,
 	stderr: process.stderr,
 });
 
-await execa('pnpm', ['--filter', 'misskey-reversi', 'build'], {
+await execa('pnpm', ['--filter', 'misskey-reversi', 'build:tsc'], {
 	cwd: _dirname + '/../',
 	stdout: process.stdout,
 	stderr: process.stderr,
 });
 
-await execa('pnpm', ['--filter', 'misskey-bubble-game', 'build'], {
+await execa('pnpm', ['--filter', 'misskey-bubble-game', 'build:tsc'], {
 	cwd: _dirname + '/../',
 	stdout: process.stdout,
 	stderr: process.stderr,
@@ -71,6 +71,12 @@ execa('pnpm', ['--filter', 'frontend', 'dev'], {
 });
 
 execa('pnpm', ['--filter', 'sw', 'watch'], {
+	cwd: _dirname + '/../',
+	stdout: process.stdout,
+	stderr: process.stderr,
+});
+
+execa('pnpm', ['--filter', 'misskey-js', 'watch'], {
 	cwd: _dirname + '/../',
 	stdout: process.stdout,
 	stderr: process.stderr,

